### PR TITLE
PL release notes for 791xx

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ For Harness on-prem releases, see [Harness Self-Managed Enterprise Edition Relea
 
 If you don't see a new feature or enhancement in your Harness account, it might be behind a Feature Flag. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
 
+### April 21, 2023, version 791xx
+
+#### What's new
+
+- Switching accounts now loads accounts on scroll. This increases the modal load time if there are more accounts to be loaded. (PL-30467)
```
Comment (brian-f-harness) [inline docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md:23]:
```suggestion
- Switching accounts now loads accounts on scroll. This increases the modal load time when there are more accounts to load. (PL-30467)
```

File: release-notes/platform.md [hunk 2]
```diff
@@ -53,14 +48,21 @@ This release does not include any early access feature.
 
 ### Fixed issues
 
-- Role assignments by **Scope** filter do not populate usernames or email addresses for existing users. (PL-32206)
+- The creation of secrets on Custom secret manager validates the existence of those secrets. (PL-31715)
```
Comment (brian-f-harness) [inline release-notes/platform.md:51]:
```suggestion
- The creation of secrets on Custom Secret Manager validates the existence of those secrets. (PL-31715)
```

File: release-notes/platform.md [hunk 3]
```diff
@@ -53,14 +48,21 @@ This release does not include any early access feature.
...
+  A code enhancement to remove secret validation during Custom Secret manager creation has fixed the issue.
```
Comment (brian-f-harness) [inline release-notes/platform.md:52]:
```suggestion
  A code enhancement to remove secret validation during Custom Secret Manager creation has fixed the issue.
```

File: release-notes/platform.md [hunk 4]
```diff
@@ -53,14 +48,21 @@ This release does not include any early access feature.
...
+
+- The tooltip for **Optional Configuration** in **Run Step** flashes in and out in loop making it difficult to read the text.(PL-32462, ZD-42201)
```
Comment (brian-f-harness) [inline release-notes/platform.md:54]:
```suggestion
- The tooltip for **Optional Configuration** in **Run Step** flashes in and out in a loop, making it difficult to read the text.(PL-32462, ZD-42201)
```

File: release-notes/platform.md [hunk 5]
```diff
@@ -72,6 +74,35 @@ This release does not include any early access feature.
 <details>
 <summary>2023 releases</summary>
 
+#### April 10, 2023, version 79015
+
+##### What's new
+- You can now navigate to the parent organization by clicking its name on a project details page. (PL-32182, ZD-41785)
```
Comment (brian-f-harness) [inline release-notes/platform.md:80]:
```suggestion
- You can now navigate to the parent organization by selecting its name on a project details page. (PL-32182, ZD-41785)
```

File: release-notes/platform.md [hunk 6]
```diff
@@ -72,6 +74,35 @@ This release does not include any early access feature.
...
+
+- Harness Git Experience now supports GitLab as a code repository. You can now select a Harness connector with any of the following Git providers to save entities in a repository: (PIE-9139)
```
Comment (brian-f-harness) [inline release-notes/platform.md:82]: Not sure if it is Harness Connector, with a capital "C"?

File: release-notes/platform.md [hunk 7]
```diff
@@ -72,6 +74,35 @@ This release does not include any early access feature.
...
+  * Github
+  * Bitbucket
+  * AzureRepo 
+  * Gitlab 
+
+- You can now open the modal in the template studio to see all applicable metadata such as a description, tags, connector, and repository name. (PIE-8692)
```
Comment (brian-f-harness) [inline release-notes/platform.md:88]:
```suggestion
- You can now open the modal in the template studio to view all applicable metadata such as a description, tags, connector, and repository name. (PIE-8692)
```

File: release-notes/platform.md [hunk 8]
```diff
@@ -72,6 +74,35 @@ This release does not include any early access feature.
...
+##### Early access
+
+This release does not include any early access feature.
+
+##### Fixed issues
+
+- Role assignments by **Scope** filter do not populate usernames or email addresses for existing users. (PL-32206)
+  A code enhancement fixed this issue.
+
+- When polling, the stage selection flickers on the pipeline execution page. (PIE-9360, ZD-42361)
```
Comment (brian-f-harness) [inline release-notes/platform.md:99]:
```suggestion
- When polling, the stage selection flickers on the **Pipeline Execution** page. (PIE-9360, ZD-42361)
```

**Review by brian-f-harness — DISMISSED** [review]
lgtm. Made a couple minor suggestions.

**Review by brian-f-harness — APPROVED** [review]
LGTM

**Review by KimberlyFields — APPROVED** [review]

**Pull request merged**